code/modules/mob/living/carbon/life.dm

```diff
@@ -9,23 +9,27 @@
 		damageoverlaytemp = 0
 		update_damage_hud()
 
-	if(stat != DEAD) //Reagent processing needs to come before breathing, to prevent edge cases.
-		handle_organs()
+	if(!IsInStasis())
+		if(stat != DEAD) //Reagent processing needs to come before breathing, to prevent edge cases.
+			handle_organs()
 
-	if(..()) //not dead
-		handle_blood()
+		if(..()) //not dead
+			handle_blood()
 
-	if(stat != DEAD)
-		handle_liver()
+		if(stat != DEAD)
+			handle_liver()
+
+		//Updates the number of stored chemicals for powers
+		handle_changeling()

```

Comment:
Worst to take a ling out of the game imo (for reference, this is under the !IsInStasis() check)

Reply:
Will revert